webapp/components/Form.tsx

```diff
@@ -68,13 +68,15 @@ const Form = (): JSX.Element => {
       <Input
         name="Navn"
         value={name}
+        pattern="^[A-Za-zÆØÅæøå\s]+$"
```

Comment:
We have concluded that we might have to deal with international names, so not sure if this would allow for e.g. Swedish and German "special" letters. But we still need it to be full names, and avoid that some people only write their first name!

Reply:
^([A-Za-zÆØÅæøåÄÖÜäöüß]+)\s([A-Za-zÆØÅæøåÄÖÜäöüß]+)$

This should match your needs now :)

Reply:
Can we throw in é, è, á and à too? To handle "Andrè/é" and Hungarian names (I know there's a Balåzs that has used this).

Might be some other cases too, but I'd say that this regez covers the most 👌

Reply:
`^([A-Za-zÆØÅæøåÄÖÜäöüßÁÉÚŐÓÍáéúőóíàèìòùÀÈÌÒÙ]+)\s([A-Za-zÆØÅæøåÄÖÜäöüßÁÉÚŐÓÍáéúőóíàèìòùÀÈÌÒÙ]+)$`

This should cover most now, i think 👌But is a bit unreadable

Reply:
Nice! I'd say it's fine 💪